bin/nightly/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "WLED",
+  "version": "0.16.0-aplha",
+  "home_assistant_domain": "wled",
+  "new_install_prompt_erase": true,
+  "builds": [
+    {
+      "chipFamily": "ESP32",
+      "parts": [
+        { "path": "esp32_bootloader_v4.bin", "offset": 0 },
```

Comment:
The regular esp32dev env is still using V3 IDF isn't it? Is the V4 bootloader backwards compatible with v3?

Reply:
didnt notice that. according to the docs it shouldnt work. 

> The bootloader does not support booting apps from older versions of ESP-IDF.

https://docs.espressif.com/projects/esp-idf/en/release-v4.3/esp32/api-guides/bootloader.html

but that bootloader has been there before my first PR which was in June 2023. 
and i used the same bootloader on the test in the nightly and it works, so it is a v4 bootloader not just a strange name?. 

since 2023 on the webinstaller. 
i am able to install the 0.14.3 release and then OTA on the v4 nightly version...